pages/linux/waydroid.md

```diff
@@ -0,0 +1,29 @@
+# waydroid
+
+> Waydroid is a container-based approach to boot a full Android system on a regular GNU/Linux system like Ubuntu.
+> Documentation: <https://docs.waydro.id/>.
+> More information: <https://waydro.id/>.
```

Comment:
```suggestion
> More information: <https://docs.waydro.id>.
```
We can directly refer to the documentation instead of the main link as it explains Waydroid in detail and goes over the install instructions too.